clang/lib/StaticAnalyzer/Checkers/WebKit/RawPtrRefMemberChecker.cpp

```diff
@@ -142,6 +144,8 @@ class RawPtrRefMemberChecker
     if (auto *ID = dyn_cast<ObjCImplementationDecl>(CD)) {
       for (auto *Ivar : ID->ivars())
         visitIvarDecl(CD, Ivar);
+      for (auto *PropImpl : ID->property_impls())
+        visitPropImpl(CD, PropImpl);
```

Comment:
`visitPropImpl` populates `IvarDeclsToIgnore`.   Should it be called before `visitIvarDecl`?

Reply:
oh, that must be why my warning is emitted in a seemingly wrong place for one of the test cases. Will fix.